embedded/__main__.py

```diff
@@ -49,9 +70,9 @@ def set_rgb_colour(led: RGBLED, colour: Colour):
 def handle_gps_polling():
     """Polls the gps lock and updates GPS_LOCKED."""
     global GPS_STATUS
-
     while not QUIT_EVENT.is_set():
         GPS_STATUS = GPS_MODULE.get_gps_status()
+        GPSLocation = GPS_MODULE.get_location()
```

Comment:
This will not set the global variable. To clarify things, always name your globals with CAPS.
If you want to assign to the global here, you need to add it to the list of globals on L72